### ISSUE TITLE
Verify bytebuddyJava task's inputs are complete

### ISSUE THREAD

**Issue by anuraaga** [open-telemetry/opentelemetry-java-instrumentation#1941]
There was some flakiness with triggering an obvious bug in the bytebuddy configuration in #1938. This might be because the inputs are not complete (perhaps missing resource files).

**Comment by mateuszrzeszutek** [comment]
I don't think that this happens anymore (after all those changes in muzzle), we can probably close this.
